src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -57,9 +57,10 @@ import { IActionViewItem } from 'vs/base/browser/ui/actionbar/actionbar';
 import { Emitter } from 'vs/base/common/event';
 import { StandardKeyboardEvent } from 'vs/base/browser/keyboardEvent';
 import { KeyCode } from 'vs/base/common/keyCodes';
+import { RedoCommand, UndoCommand } from 'vs/editor/browser/editorExtensions';
 
 export const NOTEBOOK_SELECTOR: string = 'notebook-component';
-
+const PRIORITY = 105;
```

Comment:
What exactly does this priority number do?

Reply:
@VasuBhog, it's the command priority. I used the same priority as the other UndoCommand/RedoCommand implementations that I saw in the codebase.
https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/contrib/notebook/browser/contrib/clipboard/notebookClipboard.ts#L55